docs/development/decision-records/2024-10-07-azure-distribution/README.md

```diff
@@ -0,0 +1,21 @@
+# Refactor of the Release Workflow
+
+## Decision
+
+We will stop to provide an azure flavoured distribution of the connector.
+
+## Rationale
+
+Two reasons are pushing toward this decision:
+- Tractus-X Connector is a reference implementation for Catena-X: so nothing should stop anybody to build and use their 
```

Comment:
I feel like the point of this first item should not be towards the freedom dataspace participants have to create their own distribution, because this is true since the beginning of the project. What is different at this point, IMO, is how easy it is to create and release a new distribution using the existing one as reference.

Reply:
This point was meant to tell the community that we don't "have to" maintain such distribution because our goal is not to provide different distributions for different needs, but to provide one (the reference one) and everyone should feel free to customize it according to their needs.
I can rephrase a little, but that's one of the reasons why

Reply:
I understood what you meant. Just don't want this to be seen as a "you are free to develop your own stuff if you need it" but rather a "it's not our intention to do it and we made it as easy as possible for you to do it"